redirects.txt

```diff
@@ -73,3 +73,38 @@ social_marketing/social_marketing.rst social_marketing/overview.rst    # social_
 project/planning/assignments.rst project/planning/forecast.rst    # assignments/* -> forecast/* (#583)
 
 livechat/livechat.rst livechat/overview.rst    # livechat/* -> overveiw/* (#601)
+
+point_of_sale/advanced/barcode.rst point_of_sale/shop/barcode.rst 												  					# advanced/* --> shop/* (#612)
+point_of_sale/advanced/multicashiers.rst point_of_sale/shop/multicashiers.rst 									  					# advanced/* --> shop/* (#612)
+point_of_sale/advanced/reprint.rst point_of_sale/shop/reprint.rst 												  					# advanced/* --> shop/* (#612)
+point_of_sale/advanced/cash_rounding.rst point_of_sale/shop/cash_rounding.rst 									  					# advanced/* --> shop/* (#612)
+
+quality/control_points.rst  quality/control/control_points.rst 													  					# /* --> control/* (#469)
+quality/alert_mo.rst  quality/alert/alert_mo.rst  																  					# /* --> alert/* (#469)
+quality/alert_transfer.rst  quality/alert/alert_transfer.rst 													  					# /* --> alert/* (#469)
+quality/quality_mo.rst  quality/checks/quality_mo.rst 														      					# /* --> checks/* (#469)
+quality/quality_transfers.rst  quality/checks/quality_transfers.rst 											  					# /* --> checks/* (#469)
+
+iot/connect.rst  iot/config/connect.rst  																		  					# /* --> config/* (#446)
+iot/pos.rst  iot/config/pos.rst 																				  					# /* --> config/* (#446)
+iot/iot_notfound.rst  iot/config/iot_notfound.rst 																  					# /* --> config/* (#446)
+iot/flash_sdcard.rst  iot/config/flash_sdcard.rst 																  					# /* --> config/* (#446)
```

Comment:
These have a double space instead of a single space.
It's not a problem, it still works perfectly 👍